.github/workflows/install-lua-sdk.yml

```diff
@@ -23,24 +23,43 @@ on:
           - '5.1'
           - '5.2'
           - '5.3'
-          - '5.4'
           - 'luajit-2.0.5'
-        default: '5.4'
+        default: '5.3'
       cpp-sdk-version:
-        description: "Version of the C++ Server-side SDK to use for building and testing."
+        description: "Version of the C++ Server-side SDK."
         required: false
         type: string
         default: 'launchdarkly-cpp-server-redis-source-v2.1.0'
+  workflow_call:
+    inputs:
+      package:
+        required: false
+        type: string
+      version:
+        required: false
+        type: string
+      lua-version:
+        required: false
+        type: string
+      cpp-sdk-version:
+        required: false
+        type: string
 
+env:
+  PACKAGE: ${{ inputs.package == null && 'launchdarkly-server-sdk' || inputs.package }}
+  VERSION: ${{ inputs.version == null && '' || inputs.version }}
+  LUA_VERSION: ${{ inputs.lua-version == null && '5.3' || inputs.lua-version }}
+  CPP_SDK_VERSION: ${{ inputs.cpp-sdk-version == null && 'launchdarkly-cpp-server-redis-source-v2.1.0' || inputs.cpp-sdk-version }}
```

Comment:
Forced to duplicate the correct defaults here and in the `workflow_dispatch` definition. Technically, we could omit them from `workflow_dispatch` but then the UI would just have blank fields, which is a bad experience.